tox.ini

```diff
@@ -11,13 +10,3 @@ commands =
 deps =
     twisted187: twisted==18.7.0
     twistedlatest: twisted>=20.3.0
-
-[testenv:qa]
-deps =
-    black
-    flake8
-    isort
-commands =
-    flake8 daphne tests
-    black --check daphne tests
-    isort --check-only --diff daphne tests
```

Comment:
Can we keep this? (I use it locally)

Reply:
I'd prefer not to have two ways of doing things that can drift. You can run `pre-commit run --all` locally for the same effect. Also these never pinned versions, and if they did, we'd then have two places to update (and pre-commit can update versions with `pre-commit autoupdate`)